docs/workshops/gerador-de-memes/README.md

```diff
@@ -450,16 +454,14 @@ Agora que terminamos de construir um site bem básico, podemos adicionar um mont
 
 Aqui vão algumas sugestões do que você pode adicionar:
 
-1. Tente adicionar um [seletor de cores](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/input/color), para que o usuário possa mudar a cor do textoCima/textoBaixo.
+1. Tente adicionar um [seletor de cores](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color), para que o usuário possa mudar a cor do textoCima/textoBaixo.
 2. Adicione o recurso arrastar e soltar, utilizando a [API de Drag and Drop do HTML5](https://web.dev/drag-and-drop/).
 3. Utilize [Flexbox](https://developer.mozilla.org/pt-BR/docs/Learn/CSS/CSS_layout/Flexbox) criar um layout bacana.
 4. Utilize uma API a fim obter modelos para memes, como o disponibilizado pela [ImgFlip](https://imgflip.com/api).
 5. Adicione botões de baixar e compartilhar para mostrar a todos seu meme incrível!
 
 ## Parte 5: O Fim
 
-Se você não criou uma conta no [repl.it](https://repl.it), o faça para que você consiga salvar essa sua maravilhosa criação!
-
 Se você está tendo problemas para se cadastrar, fale com um líder de seu clube ou com alguém no [Slack do Hack Club](https://hackclub.com/slack)!
```

Comment:
Colocar este texto na Parte 2 do documento, onde será pedido ao usuário criar uma conta no repl.it

Reply:
É melhor tmb colocar o link da comunidade no discord, assim a pessoa pode falar diretamente com a gente

Reply:
É melhor também colocar a comunidade do discord para que a pessoa possa falar diretamente com a gente